.circleci/config.yml

```diff
@@ -94,10 +164,24 @@ workflows:
                 - staging
                 - release
       - push:
+          context: twreporter
           requires:
             - build
           filters:
             branches:
               only:
                 - staging
                 - release
+      - slack/on-hold: 
+          context: twreporter
+          requires:
+            - push
+      - wait_for_approval:
+          requires:
+            - push
```

Comment:
Do we need `push` here ?
`slack/on-hold` requires `push` already.

Reply:
I believe we could leave the `slack/on-hold` job for dependency.

Reply:
Would you like to explain more?

Reply:
We can prune the push job dependency since the slack/on-hold job already has that dependency. By definition of the requires, the push job can be saved here.

Reply:
What do you mean `the push job can be saved here`?

My suggestion is that we don't need `push` here, since `slack/on-hold` only requires `push` already.

However, I am ok for current config.

Reply:
What do you mean `the push job can be saved here`?
My suggestion is that we don't need `push` here, since `slack/on-hold` requires push already.
However, I am ok for current config.

Reply:
Push job can be saved here =>  push job 可以被省略，我從上一個comment的意思就是我可以拿掉，如果英文容易造成誤會，那我想之後還是用中文吧

Reply:
好喔，用中文似乎可以減輕溝通成本。